apps/src/storage/dataBrowser/ConfirmDeleteButton.story.jsx

```diff
@@ -2,21 +2,17 @@ import React from 'react';
 import ConfirmDeleteButton from './ConfirmDeleteButton';
 import {action} from '@storybook/addon-actions';
 
-export default storybook => {
-  return storybook
-    .storiesOf('Buttons/ConfirmDeleteButton', module)
-    .addStoryTable([
-      {
-        name: 'basic example',
-        description: ``,
-        story: () => (
-          <ConfirmDeleteButton
-            title="Delete table?"
-            body="Are you sure you want to delete the table?"
-            buttonText="Delete table"
-            onConfirmDelete={action('delete table')}
-          />
-        )
-      }
-    ]);
+export default {
+  title: 'ConfirmDeleteButton',
+  component: ConfirmDeleteButton
+};
+
+const Template = args => <ConfirmDeleteButton {...args} />;
+
+export const BasicExample = Template.bind({});
+BasicExample.args = {
```

Comment:
For a scenario like this where there's only only story, is it better to list the args separate like this or to just include them in the template itself?

Reply:
Good question, my thought was to keep it separate to make it easier to add more stories in the future, but I am open to suggestion.

Reply:
I had the same question. I had 2 files with just one story each and included the args in the template. I'm good either way. @madelynkasula do you have a preference?

Reply:
I'm also listening in to this conversation. I went with Molly's route for consistency and expandability.

Reply:
> my thought was to keep it separate to make it easier to add more stories in the future

+1 to molly's comment. if the props are something that _all_ stories will want (e.g., the `isOpen` prop for a dialog will always be true so we can render the dialog in every story), i hard-code those args in the template. otherwise, i keep them in individual stories.

i lean toward the latter (keeping props in individual stories) because it's easier to move these props into the template later than it is to do the reverse refactor

Reply:
Thanks @madelynkasula ! I was also wondering about naming convention for a component with only story - I see default was used sometimes in the old Storybook version, or the name of the component itself. I ended up using BasicExample but someone also use Basic. Is there a preference?

Reply:
hmmmm, that's a good question, alice. i think using the name of the component is redundant since all stories are nested under the component name. otherwise, it doesn't really matter, but we should set a convention to be consistent! let's go with "BasicExample" unless others have opinions